components/x-article-save-button/src/ArticleSaveButton.scss

```diff
@@ -0,0 +1,41 @@
+@import 'o-icons/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
+$icon-size: 38px;
+
+@mixin background-icon($name, $color) {
+	@include oIconsGetIcon($name, oColorsGetPaletteColor($color), $icon-size, $iconset-version: 1);
+	display: block;
+	margin-bottom: -5px;
+}
+
+.root {
+	display: inline-block;
+}
+
+.button {
+	@include oTypographySans($scale: -2);
+	border: 0;
+	width: $icon-size;
+	padding: 0;
+	color: oColorsGetPaletteColor('black');
+	background-color: transparent;
+	text-align: center;
+
+	&:focus {
+		outline: none;
+	}
+
+	.icon {
+		@include background-icon(bookmark-outline, 'black');
+	}
+
+	body:not(:global(.touch-device)) &:not(:focus):hover .icon {
```

Comment:
Worth a comment mentioning what this is?

Reply:
Is `.touch-device` an `n-ui` thing? I'd either document that the component expects this class on touchscreens, or consider `@media (hover: hover)` which is [well-supported](https://caniuse.com/#feat=css-media-interaction) now